k8s/deployment.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: apps/v1
```

YAMLlint (1.37.1)

[error] wrong new line character: expected \n

(new-lines)